packages/discussions/graphql/resolvers/Comment.js

```diff
@@ -138,6 +138,16 @@ module.exports = {
   text: (comment, args, context) =>
     textForComment(comment, false, context),
 
+  featuredContent: ({ published, adminUnpublished, featuredAt, featuredContent }) =>
```

Comment:
Currently all featured comment text is plain text right now like `comment.preview`. I would prefer to remove the `featuredContent` resolver and only provide `featuredText` for now.